src/jdk.compiler/share/classes/com/sun/tools/javac/main/JavacToolProvider.java

```diff
@@ -25,7 +25,9 @@
 
 package com.sun.tools.javac.main;
 
+import com.sun.tools.javac.util.JavacMessages;
 import java.io.PrintWriter;
+import java.util.Optional;
 import java.util.spi.ToolProvider;
```

Comment:
at least in javac, we normally sort `java.*` and `javax.*` imports before other imports.

Reply:
Sorted imports accordingly.